tests/expectations/core/test_expect_column_values_to_be_in_type_list.py

```diff
@@ -30,7 +30,7 @@ def test_expect_column_values_to_be_in_type_list_dialect_pyathena_string(sa):
     validator.execution_engine.dialect_module = aws.sqlalchemy_athena
 
     result = validator.expect_column_values_to_be_in_type_list(
-        "col", type_list=["string", "boolean"]
+        "col", type_list=["string", "boolean", "TEXT"]
```

Comment:
This addition is because the call to `compile` now maps the class `TEXT` to the string `"TEXT"`?